js/packages/web/src/views/auction/index.tsx

```diff
@@ -243,26 +245,25 @@ export const AuctionView = () => {
                   : edition}
               </span>
             </div>
-            <div className="info-item-wrapper">
-              <span className="item-title">Winners</span>
-              <span>
-                {winnerCount === undefined ? (
-                  <Skeleton paragraph={{ rows: 0 }} />
-                ) : (
-                  winnerCount
-                )}
-              </span>
-            </div>
-            <div className="info-item-wrapper">
-              <span className="item-title">NFTs</span>
-              <span>
-                {nftCount === undefined ? (
-                  <Skeleton paragraph={{ rows: 0 }} />
-                ) : (
-                  nftCount
-                )}
-              </span>
-            </div>
+            {(auction?.items.length || 0) > 1 && (
+              <div className="info-item-wrapper">
+                <span className="item-title">Remaining</span>
+                <span>
+                  {art.maxSupply === undefined ? (
+                    <Skeleton paragraph={{ rows: 0 }} />
+                  ) : (
+                    <span>
+                      {`${(art.maxSupply || 0) - (art.supply || 0)} of ${
+                        art.maxSupply || 0
+                      } `}
```

Comment:
Should this be based on the supply of NFT or the number of winners for the listing? I think the second is more appropriate.

In this case the maxSupply should be replaced with the auction.numberOfWinners and supply set to the number of bids. If bids is > maxWinners then sold out.

Reply:
No, it takes total NFTs in account not during current listing. I sold out it will change it's status

Reply:
This is status of all supply

Reply:
Basically this is relevant info for the end user/collector

Reply:
The previous tile was to signal how many winners there will be for the auction. How can showcase both cleanly? 

Sounds like both the supply of the listing and the supply of the NFT are valuable to know as a collector checking out the listing.

Reply:
Okay I'll bring them back. You're right. Also I will check how five items fit

Reply:
![image](https://user-images.githubusercontent.com/7579297/148996453-a1a1ecf2-ffde-4e46-9768-14ae3cc87c03.png)
Looks great!